score.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2016 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
Just curious, do we have to keep adding these headers?

Reply:
I do think we should have the Apache header on all files, since this is standard practice when a repo is licensed a certain way. But it does feel weird to be adding 

```
// Copyright 2016 Google Inc. All rights reserved.
```

I made a personal TODO to ping the legal team about this.